### PR TITLE
Release notes for 0.13.1, plus an update to the list of authors.

### PR DESCRIPTION
**Pull request by apdavison**
These notes assume that #1338 gets merged before the release. If it doesn't get merged, remove the mention of BlackrockIO.

Note that I added affiliations for new contributors where I could find them. If you see yourself in the author list and there's no affiliation given, please let us know in the comments if you'd like your affiliation to be added. If there's an error in your affiliation, please also let us know.

### CODE REVIEW

**Comment by alejoe91** [comment]
@apdavison can we wait for this https://github.com/NeuralEnsemble/python-neo/pull/1457 too?

**Comment by apdavison** [comment]
sure. I was planning to make the release at the end of the afternoon tomorrow.

**Comment by zm711** [comment]
Perfect, both the PRs have tests passing. @samuelgarcia had mentioned that he was going to look at #1338 previously so not sure if he wants to give that a look-over before or not. I took a quick look at it and it seemed fine, but I'm not super familiar with BlackRock. #1457 should be ready for merge as well by tomorrow morning. So sounds good to me. 

Release notes look good to me.

**Comment by samuelgarcia** [comment]
@zm711 : #1338 is OK for me. merci.

File: doc/source/authors.rst [hunk 1]
```diff
@@ -108,6 +112,16 @@ and may not be the current affiliation of a contributor.
 26. Georgia Institute of Technology
 27. Emory University
 28. Harvard Medical School
+29. Institute for Advanced Simulation (IAS-6), Forschungszentrum Jülich, Germany
```
Comment (alejoe91) [inline doc/source/authors.rst:115]: @apdavison could you change my affiliation 23. to "Allen Institute for Neural Dynamics, Seattle, WA, United States"
?

Comment (apdavison) [reply doc/source/authors.rst:115]: Is affiliation 23 incorrect (i.e. you never worked there, or didn't contribute to Neo while you worked there), or is it that you moved?

If it's just that you moved, then I would add the Allen Institute as a second affiliation.

Comment (apdavison) [reply doc/source/authors.rst:115]: and I guess we should add CatalystNeuro as a third affiliation in that case?

Comment (alejoe91) [reply doc/source/authors.rst:115]: Hi @apdavison 

Yeah the institutution is correct, but I wonder whether we should keep all past institutions...What do you think?
IMO, it's better to keep the current institution(s), which for me are Allen Institute + CatalystNeuro

Comment (apdavison) [reply doc/source/authors.rst:115]: Past institutions are potential copyright co-owners, so we need to keep track of them.

Comment (zm711) [reply doc/source/authors.rst:115]: If that's the case then I should probably add a second affiliation (I can do a PR myself soon). My MD/PhD will be from Harvard but my current lab is at Massachusetts General Hospital.

**Comment by zm711** [comment]
Thanks all!

**Pull request merged**